src/main/resources/io/jenkins/plugins/folderauth/FolderAuthorizationStrategyManagementLink/index.jelly

```diff
@@ -1,309 +1,10 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout">
-    <l:layout permission="${app.ADMINISTER}" norefresh="true" title="${%title}">
+    <l:layout permission="${app.ADMINISTER}" norefresh="true" title="${%title}" nogrid="true">
```

Comment:
That works (with some hacks) but the sidebar breaks when `nogrid="true"`.

![image](https://user-images.githubusercontent.com/11471599/83107244-8487dd80-a0db-11ea-9fe4-20456cdab136.png)

Would it make sense to remove the sidebar?

Reply:
@uhafner thoughts? I'm guessing there's some minimal styles you can add to fix the sidebar, or the sidebar shouldn't be relying on those styles and should be fixed in Jenkins core, cc @fqueiruga

Reply:
Which Jenkins version are you using? It still shows the old UI. Maybe that is the problem? I am already using the `nogrid` flag and the sidebar does not break.

Reply:
When you use something that loads Bootstrap 4 then you also need jquery3, which leads to some additional problems. I needed to add the following fix on all my pages:
https://github.com/jenkinsci/bootstrap4-api-plugin/blob/master/src/main/webapp/js/no-prototype.js

Reply:
@uhafner Jenkins 2.235


> When you use something that loads Bootstrap 4 then you also need jquery3, which leads to some additional problems. I needed to add the following fix on all my pages

I'm using [react-bootstrap](https://react-bootstrap.github.io/) which does not need jquery. Quoting from their website:

> React-Bootstrap replaces the Bootstrap JavaScript. Each component has been built from scratch as a true React component, without unneeded dependencies like jQuery.

Reply:
@uhafner is there any way to load the bootstrap layout just for the CSS without loading the bootstrap JS? That would be a huge win for many plugins.

Reply:
Yes, that should be possible. Currently 
```
<st:adjunct includes="io.jenkins.plugins.bootstrap4"/>
``` 
includes both. But you can already include it using 
```
<link type="text/css" rel="stylesheet" href="${resURL}/plugin/bootstrap4-api/css/bootstrap.min.css"/>
```

The. problem is: you don't know which components require JS and which not.

Reply:
> @uhafner Jenkins 2.235
> 

Maybe the problem is, that if you do not load the BS3 grid then you rather need to load the BS4 grid. So in my layouts I am using `nogrid` *and* BS4 grid. If you do not load a grid it might break things.